src/components/LuxLibraryHeader.vue

```diff
@@ -186,7 +186,7 @@ export default {
   @media (min-width: 900px) {
     .full-name {
       display: block;
-      font-size: var(--font-size-large);
+      font-size: var(--font-size-base);
```

Comment:
To me, having the app name the same size as the rest of the text looks odd

![the header of the catalog, with the catalog looking a bit smaller than the menu items](https://github.com/user-attachments/assets/c419e01a-5f7a-460b-8cd5-102df3ca9fc6)

Reply:
thanks Jane I didnt want to change this one.